recipes/yaqd-attune/meta.yaml

```diff
@@ -0,0 +1,78 @@
+{% set name = "yaqd-attune" %}
+{% set version = "2022.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/yaqd-attune-{{ version }}.tar.gz
+  sha256: 535922d16698edb08fcd7a88f8c50dd0e7872019303f01f08ca86e69be4f96b0
+
+build:
+  number: 0
+
+test:
+  imports:
+    - yaqd_attune
+  commands:
+    - pip check
+  requires:
+    - pip
+
+outputs:
+  - name: yaqd-attune
+    build:
+      noarch: python
+      script: python -m pip install . -vv
```

Comment:
The reason that `{{ PYTHON }}` doesn't work here is that script in the output section expects the name of a script not a script literal.

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#specifying-files-to-include-in-output